src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -123,3 +125,35 @@ def get(self, request: Request, organization: Organization) -> HttpResponse:
                 "chunk_ids": [el["chunk_id"] for el in chunk_ids],
             },
         )
+
+
+@region_silo_endpoint
+class OrganizationProfilingChunksFlamegraphEndpoint(OrganizationProfilingBaseEndpoint):
+    def get(self, request: Request, organization: Organization) -> HttpResponse:
+        if not features.has("organizations:profiling", organization, actor=request.user):
+            return Response(status=404)
+
+        params = self.get_snuba_params(request, organization, check_global_views=False)
```

Comment:
Why is this turning global views off here

Reply:
honestly, this was just carried over from the other 2 endpoints defined in `organization_profiling_profiles.py`, which also define `check_global_views=False`. Should the default (`True`) be used instead?